recipe/meta.yaml

```
@@ -29,7 +29,7 @@ requirements:
     - jupyter_core
     - jupyter_client >=4.1
     - pygments
-    - pyqt  # not a dependency listed in setup.py but is still a runtime requirement for qtconsole https://qtconsole.readthedocs.io/en/stable/installation.html#installing-qt-if-needed
+    - pyqt ==5.* # not a dependency listed in setup.py but is still a runtime requirement for qtconsole https://qtconsole.readthedocs.io/en/stable/installation.html#installing-qt-if-needed
```

```suggestion
    - pyqt # not a dependency listed in setup.py but is still a runtime requirement for qtconsole https://qtconsole.readthedocs.io/en/stable/installation.html#installing-qt-if-needed
```
According to this PR, qtconsole does support pyqt6
https://github.com/jupyter/qtconsole/pull/525

---

You can try adjusting the test to make them pass with pyqt 6, similar to https://github.com/AnacondaRecipes/superqt-feedstock/blob/main/recipe/meta.yaml#L35-L55

Or add pyqt 5 to test/requires

---

`QT_QPA_PLATFORM` doesn't work, but I've tested locally and everything is fine with pyqt 6, so I've just set it to 5 in tests.